src/status_im/ui/components/list_item/styles.cljs

```diff
@@ -1,24 +1,86 @@
 (ns status-im.ui.components.list-item.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
   (:require [status-im.ui.components.colors :as colors]))
 
-(defn container [small?]
-  {:height           (if small? 52 64)
+(defn container [type]
+  {:min-height         (cond
+                         (= type :small)          52
+                         (= type :section-header) 40
+                         :else                    64)
+   :flex               1
+   :flex-direction     :row
+   ;; :background-color   colors/red-transparent-10
+   :justify-content    :flex-start
+   :padding-horizontal 16
+   :padding-top        (cond
+                         (= type :default) 10
+                         :else             14)
+   :padding-bottom     (cond
+                         (= type :small)          14
+                         (= type :section-header) 4
+                         :else                    10)
+   :align-items        :center})
+
+(defnstyle icon-container [color]
+  {:border-radius    20
+   :width            40
+   :height           40
    :align-items      :center
-   :flex-direction   :row
-   :background-color :white})
+   :justify-content  :center
+   :background-color (or color colors/blue-light)})
+
+(defn icon [color]
+  {:color     (or color colors/blue)
+   :font-size 16})
 
-(defn title [small? subtitle]
-  (merge (when-not small?
-           {:typography :title})
-         (when subtitle
+(defn title-prefix [type icon? subtitle content]
+  (merge {:margin-left (if icon? 2 0)
+          ;; :background-color colors/red-transparent-10
+          :line-height 22
+          :align-self  :stretch}
+         (when (= type :default)
+           {:typography  :title
+            :height      20
+            :line-height 20})
+         (when (or subtitle content)
            {:typography :main-medium})))
 
-(def subtitle
-  {:margin-top  4
-   :color       colors/gray})
+(defn title [type icon? title-prefix subtitle content title-row-accessory]
+  (merge {:margin-left (if icon? 2 0)
+          :flex        1
+          :flex-grow   1
+          ;; :background-color colors/red-transparent-10
+          :flex-shrink 1
+          :line-height 22
+          :align-self  :stretch}
+         (when (= type :default)
+           {:typography  :title
+            :height      20
+            :line-height 20})
+         (when title-prefix
+           {:margin-left (if (string? title-prefix) 0 2)})
+         (when title-row-accessory
+           {:margin-right 16})
+         (when (or subtitle content)
+           {:typography :main-medium})))
+
+(defn subtitle [icon? subtitle-row-accessory]
+  (cond-> {:margin-left (if icon? 2 0)
+           :flex        1
+           :flex-grow   1
+           :flex-shrink 1
+           :align-self  :stretch
+           :font-size   15
+           :text-align  :left
+           ;; :background-color colors/red-transparent-10
+           :line-height 22
```

Comment:
you should never have to provide line-height it is always computed based on font size

Reply:
Understood, but, in this case, the figma design spec is explicit about it, and following it resulted in near-about pixel perfectness. So, had to do it.

Reply:
please add a comment next to the line-height that you had to overwrite for pixel perfectness, otherwise I or someone else will delete them thinking it's a mistake in the future